examples/TLS_DenseSkOp.cpp

```diff
@@ -0,0 +1,164 @@
+    }
+    int64_t d = 1;       // Number of columns of B
+    int64_t sk_dim = 2*(n+d);
+
+    // Initialize workspace
+    double *AB = new double[m*(n + d)]; // Store [A B] in column major format
+    double *SAB = new double[sk_dim*(n+d)];
+    double *X = new double[n];
+    double *res = new double[n];
+
+    // Initialize workspace for the sketched svd 
+    double *U = new double[sk_dim*sk_dim];
+    double *svals = new double[n+d];
+    double *VT = new double[(n+d)*(n+d)];
+
+    // Initialize noisy gaussian data
+    init_noisy_data(m, n, d, AB);
+
+    // Define properties of the sketching operator
+
+    // Initialize seed for random number generation
+    uint32_t seed = 0;
+
+    // Define the dense distribution that the sketching operator will sample from
+    /* Additional dense distributions: RandBLAS::DenseDistName::Uniform - entries are iid drawn uniform [-1,1]
+     *                                 RandBLAS::DenseDistName::BlackBox - entires are user provided through a buffer
+    */
+    auto time_constructsketch1 = high_resolution_clock::now();
+    RandBLAS::DenseDistName dn = RandBLAS::DenseDistName::Gaussian;
+    
+    // Initialize dense distribution struct for the sketching operator
+    RandBLAS::DenseDist Dist(sk_dim,   // Number of rows of the sketching operator 
+                             m,        // Number of columns of the sketching operator
+                             dn);     // Distribution of the entires
+
+    //Construct the dense sketching operator
+    RandBLAS::DenseSkOp<double> S(Dist, seed);  
+    RandBLAS::fill_dense(S);
+    auto time_constructsketch2 = high_resolution_clock::now();
+
+    // Sketch AB
+    // SAB = alpha * \op(S) * \op(AB) + beta * SAB
+    auto time_sketch1 = high_resolution_clock::now();
+    RandBLAS::sketch_general<double>(
+            blas::Layout::ColMajor,    // Matrix storage layout of AB and SAB
+            blas::Op::NoTrans,         // NoTrans => \op(S) = S, Trans => \op(S) = S^T
+            blas::Op::NoTrans,         // NoTrans => \op(AB) = AB, Trans => \op(AB) = AB^T
+            sk_dim,                    // Number of rows of S and SAB
+            n+d,                       // Number of columns of AB and SAB
+            m,                         // Number of rows of AB and columns of S
+            1,                         // Scalar alpha - if alpha is zero AB is not accessed
+            S,                         // A DenseSkOp or SparseSkOp sketching operator
+            0,                         // Row offset of S
+            0,                         // Column offset of S
+            AB,                        // Matrix to be sketched
+            m,                         // Leading dimension of AB
+            0,                         // Scalar beta - if beta is zero SAB is not accessed
+            SAB,                       // Sketched matrix SAB
+            sk_dim                     // Leading dimension of SAB
+    );
```

Comment:
You can remove the row and column offsets of S. We have an overload of ``sketch_general`` that automatically sets them to 0 if not provided.